src/Bootstrap.php

```diff
@@ -36,35 +36,37 @@ public function bootstrap($app)
             ];
         }
 
-        $parameter = null;
-        if (!Env::get('BI_DB_DSN')) {
-            $parameter = 'BI_DB_DSN';
-        } elseif (!Env::get('BI_DB_USERNAME')) {
-            $parameter = 'BI_DB_USERNAME';
-        } elseif (!Env::get('BI_DB_PASSWORD')) {
-            $parameter = 'BI_DB_PASSWORD';
-        }
-        if ($parameter) {
-            throw new InvalidConfigException(Yii::t('biDashboard', 'The {env_parameter} parameter is not set, add the parameter in the env file of the project.', ['env_parameter' => $parameter]));
+        if (!Yii::$app->has('biDB')) {
```

Comment:
Why do you need to check **biDB** component exist? Is there any problem on set env's?

Reply:
because any project maybe has some changes in its mysql connection, so we it can set own connection then the modules connection shoudn't be used anymore.

Reply:
and as I remember this module hasn't used until now, I think its better to change old migration but yes if it is using by any project we can changed them as new migrations